docs/userGuide/boilerplate.md

```diff
@@ -0,0 +1,24 @@
+The `_markbind/boilerplates` folder in the site's root directory can be used to store repetitive files. You might have a file `structure.md` (that contain text such as `<include src="text.md">`) that needs to be used in many places. You want the `text.md` to include to depend on the folder `structure.md` is supposed to be in (in this case, `book/architecture/full.md`). Instead of duplicating the file in different locations, you will only need a copy in the boilerplate directory - the final outcome is as if the file was present in the different locations.
+
+To specify a boilerplate file, add `boilerplate` in the attribute for `include` and `dynamic-panel`.
+
+E.g. In `book/architecture/architecturalStyles/index.md`,
+```html
+  <dynamic-panel src="structure.md" boilerplate />
+  <include src="../structure.md" boilerplate />
```

Comment:
Move `<include />` before `<dynamic-panel />` since most of the examples use it and it's mentioned first.